sdk/java-sdk/src/main/scala/kalix/javasdk/impl/ProxyInfoHolder.scala

```diff
@@ -55,23 +56,24 @@ class ProxyInfoHolder(system: ExtendedActorSystem) extends Extension {
         proxyInfo.internalProxyHostname
       }
 
-    this._proxyHostname = Some(chosenProxyName)
-
-    this._identificationInfo = proxyInfo.identificationInfo
+    // don't set if already overridden (by testkit)
+    _proxyHostname.compareAndSet(null, chosenProxyName)
+    _proxyPort.compareAndSet(-1, proxyInfo.proxyPort)
+    _identificationInfo = proxyInfo.identificationInfo
```

Comment:
check here below, line 65

We are loning `proxyInfo`, we should log `proxyPort`

Reply:
Done in [56db155](https://github.com/lightbend/kalix-jvm-sdk/pull/1282/commits/56db15519bb3eda42b05cb69ca1040c485552056)